projects/packages/assets/tests/php/test-assets.php

```diff
@@ -688,6 +688,36 @@ public static function provide_register_script() {
 					'is_rtl' => false,
 				),
 			),
+			'Strategy Defer'                             => array(
+				array( 'single-file', 'test-assets-files/single-js-file.js', __FILE__, array( 'strategy' => 'defer' ) ),
+				array(
+					'wp_register_script' => array(
+						'single-file',
+						"$url_base/single-js-file.js?minify=false",
+						array(),
+						2883865438,
+						array(
+							'strategy' => 'defer',
+						),
+					),
+					array(),
+				),
+			),
+			'Strategy Async'                             => array(
+				array( 'single-file', 'test-assets-files/single-js-file.js', __FILE__, array( 'strategy' => 'async' ) ),
+				array(
+					'wp_register_script' => array(
+						'single-file',
+						"$url_base/single-js-file.js?minify=false",
+						array(),
+						2883865438,
+						array(
+							'strategy' => 'async',
+						),
+					),
+					array(),
+				),
```

Comment:
Same deal as above. I'll make the suggestion here too in case that's what you intended.
```suggestion
				),
				array(),
```